### ISSUE TITLE
Package "github.com/go-kit/kit/log" is deprecated: Use "github.com/go-kit/log" instead

### ISSUE THREAD

**Issue by anishbista60** [kubevirt/kubevirt#12955]
Is your feature request related to a problem? Please describe:
In the code we used `"github.com/go-kit/kit/log"` which has been deprecated.

Describe the solution you'd like:
Replace  `"github.com/go-kit/kit/log"` with `"github.com/go-kit/log"` .

[Deprecation notice](https://github.com/go-kit/kit/blob/master/log/README.md)

**Comment by iholder101** [comment]
@anishbista60 thanks very much for reporting this!
Can you add a link to the issue description about notifying that this package is indeed deprecated?

**Comment by orelmisan** [comment]
I suggest leveraging this deprecation to use a single logging solution in KubeVirt's code base.

**Comment by anishbista60** [comment]
> @anishbista60 thanks very much for reporting this! Can you add a link to the issue description about notifying that this package is indeed deprecated?

I have updated. you can check.

**Comment by iholder101** [comment]
> > @anishbista60 thanks very much for reporting this! Can you add a link to the issue description about notifying that this package is indeed deprecated?
> 
> I have updated. you can check.

Thank you!
Copying it to here as well for visibility:
> Deprecation notice: The core Go kit log packages (log, log/level, log/term, and log/syslog) have been moved to their own repository at github.com/go-kit/log. The corresponding packages in this directory remain for backwards compatibility. Their types alias the types and their functions call the functions provided by the new repository. Using either import path should be equivalent. Prefer the new import path when practical.


/label good-first-issue

**Comment by iholder101** [comment]
@dosu can you remove the "area/launcher" label?

**Comment by Arpit529Srivastava** [comment]
please assign this issue to me and label it as hactoberfest

**Comment by anishbista60** [comment]
@orelmisan IMO kubevirt is not taking part in hacktoberfest this year. @iholder101 @alicefr please take a look.

**Comment by alicefr** [comment]
Hi Anish,

I haven't heard anything about it this year. cc @Andrew Burden
***@***.***>

**Comment by anishbista60** [comment]
> about

previously, i also pinged many time in slack. Seems ! we are not taking part.

**Comment by anishbista60** [comment]
> Hi Anish, I haven't heard anything about it this year. cc @aburdenthehand any new about hacktoberfest?

previously, i also pinged many time in slack. Seems ! we are not taking part.

**Comment by dhiller** [comment]
@anishbista60 these are the projects that are taking part in Hacktoberfest this year: https://github.com/search?q=org%3Akubevirt%20topic%3Ahacktoberfest&type=repositories

So, yes, k/kubevirt is also taking part.

**Comment by dhiller** [comment]
To quote
> * Pull/merge requests can be made in any [GitHub](https://github.com/topics/hacktoberfest) or [GitLab](https://go.gitlab.com/ubCLKL) hosted project that’s participating in Hacktoberfest (look for the “hacktoberfest” topic)

[source](https://hacktoberfest.com/participation/)

**Comment by dhiller** [comment]
> area



> please assign this issue to me and label it as hactoberfest

Labelling it is not required - if you create any PR for this repo, it will automatically count, provided it satisfies the requirements for Hacktoberfest.

**Comment by sumanjeet0012** [comment]
I want to work on this issue.

**Comment by Ashlok2003** [comment]
@alicefr @iholder101  I would like to work on this issue. Can you share more details? Also, is this PR still open?

**Comment by iholder101** [comment]
@Ashlok2003 I see closed PR from a few months ago, but it seems nothing were merged.

@Arpit529Srivastava are you still planning to work on this?

If not, @Ashlok2003, I suggest to use comments like [this one](https://github.com/kubevirt/kubevirt/pull/13009#issuecomment-2394866451) as guidance. In general, the closed PR contains a discussion that is extremely helpful for implementing this.

Good luck! Let us know if you need any help.

**Comment by Ashlok2003** [comment]
@iholder101  I attempted this PR while considering the highlighted closed issue you mentioned. The problem was that they were unable to properly run make generate and were facing issues. However, I can run it without any problems. But when I do, some files also get modified. Should I include them or only update the beneficial files, leaving out those modified by Bazel? 

![Image](https://github.com/user-attachments/assets/b8972e56-ccbe-4f21-9101-48032f056fae)

**Comment by iholder101** [comment]
> Should I include them or only update the beneficial files, leaving out those modified by Bazel?

Yes, bazel files need to merge alongside your code changes. These should be auto-created by executing `make`, or `make fmt` (the quicker version, but that won't build the entire project).

Also, as mentioned in [this comment](https://github.com/kubevirt/kubevirt/pull/13009#pullrequestreview-2350967355), after modifying `go.mod` you'd likely be needing to run `make deps-update` and `make && make generate`.

I think that in general it would be easier if you'd create a PR then we can see the actual changes you did and review them.

**Comment by Ashlok2003** [comment]
I've run `make deps-update` and `make generate` as you suggested. Also, I've executed `make` to ensure everything builds correctly. 
I'll proceed to create a PR so you can review the changes.
